b9lab-content/2-what-is-cosmos/02-blockchain-and-cosmos.md

```diff
@@ -0,0 +1,154 @@
+
+</ExpansionPanel>
+
+Finally, when developing on Ethereum, the developer needs to contend with **two layers of governance**: The chain's governance and the application's governance. Whatever the DApp developers wish for their application's governance, they must contend with the underlying chain's governance.
+
+Given the features of existing public blockchain projects and the requirements for privacy in certain industries, a push started in favor of **private, or managed, chains**, and more generally _private distributed ledgers_, i.e. blockchains with access barriers and sophisticated permission management. Platforms for permissioned networks such as R3's Corda and the Hyperledger Project's Hyperledger Fabric from the Linux Foundation are such examples.
+
+The eventual development of further and more complex applications required a more flexible environment. This led to the launch of multiple **purpose-built chains**, each providing a platform tailored to the necessities of its application, i.e. use case. Still, each of these blockchains acted as self-contained environments, limited by the use cases they were envisioned for.
+
+**General-purpose chains are limited to simplistic use case applications, while purpose-built chains only fit certain use cases.** The question arises: *Is it possible to build a platform for all use cases that does away with the limitations of general-purpose chains?*
+
+## How does Cosmos fit into the general development of blockchain technology?
+
+In 2014, Jae Kwon and Ethan Buchman founded the Cosmos network with its consensus algorithm, [Tendermint](https://tendermint.com/).
+
+<ExpansionPanel title="The skinny on Tendermint">
+
+Tendermint is a BFT consensus algorithm and a consensus engine. It is a Replicated State Machine (RSM) that helps replicating an application on several machines. As such, it guarantees BFT properties for distributed systems and their applications.
+
+It does this in:
+
+* A **secure** manner - Tendermint continues working even if up to 1/3 of machines fail, and
+* A **consistent** way - every machine computes the same state and accesses the same transaction log.
+
+Tendermint is widely used across the industry and the most mature BFT consensus engine for Proof-of-Stake (PoS) blockchains.
+
+For more on Tendermint, have a look at this helpful [introduction](https://docs.tendermint.com/master/introduction/what-is-tendermint.html).
+
+</ExpansionPanel>
+
+Initially, Cosmos was an open-source community project built by the Tendermint team. Since then, the **Interchain Foundation** (ICF) has assisted with the development and launch of the network. The ICF is a Swiss non-profit that after an initial coin offering (ICO) in 2017 raised funds to finance the development of open-source projects building on the Cosmos network.
```

Comment:
```suggestion
Initially, Cosmos was an open-source community project built by the Tendermint team. Since then, the **Interchain Foundation** (ICF) has assisted with the development and launch of the network. The ICF is a Swiss non-profit that after an initial fundraising event in 2017 raised funds to finance the development of open-source projects building on the Cosmos network.
```